### PR TITLE
Deprecate ft.add, ft.del, ft.drop

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

**Comment by ashtul** [comment]
fix #1457 
Should we move `ft.get` `ft.mget` and `ft.synadd` to end of file

**Comment by MeirShpilraien** [comment]
> fix #1457
> Should we move `ft.get` `ft.mget` and `ft.synadd` to end of file

Good point

**Comment by gkorland** [comment]
ref #1453

File: docs/Commands.md [hunk 1]
```diff
@@ -168,140 +168,6 @@ O(1)
 ### Returns
 OK or an error
 
----
-
-## HSET/HSETNX/HDEL/HINCRBY/HDECRBY
```
Comment (ashtul) [inline docs/Commands.md:173]: Shouldn't we have a section about how to use HSET etc?

**Review by ashtul — COMMENTED** [review]
Looks good.
We do have to figure out to place info about using the hash commands

**Comment by ashtul** [comment]
fix #1454 
fix #1450

**Review by rafie — APPROVED** [review]

**Comment by ashtul** [comment]
fix #1486

**Pull request merged**